cobalt/layout/topmost_event_target.cc

```diff
@@ -471,8 +477,10 @@ void TopmostEventTarget::ConsiderElement(dom::Element* element,
   if (!element) return;
   math::Vector2dF element_coordinate(coordinate);
   LayoutBoxes* layout_boxes = GetLayoutBoxesIfNotEmpty(element);
-  bool consider_element_and_children = ShouldConsiderElementAndChildren(
-      element, &element_coordinate, consider_only_fixed_elements);
+  bool consider_element_and_children =
+      layout_boxes &&
+      ShouldConsiderElementAndChildren(element, &element_coordinate,
```

Comment:
It looks to me like ShouldConsiderElementAndChildren already handles when layout_boxes is null?

Reply:
Okay, I don't have issues with it if it's not coming to chrobalt